requirements.txt

```diff
@@ -19,13 +19,18 @@ sacrebleu==1.5.1
 seqeval==1.2.2
 
 # for utility
-tqdm
+tqdm~=4.49.0
```

Comment:
This needs to be moved into the transformations folder.

Reply:
Excluded requirements.txt from this PR